samples/beta/cancel_operation.js

```diff
@@ -0,0 +1,61 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+async function main(
+  operationFullId = 'projects/YOUR_PROJECT_ID/locations/YOUR_LOCATIOIN/operations/OPERATION_ID'
+) {
+  // [START automl_cancel_operation_beta]
+
+  /**
+   * TODO(developer): Uncomment these variables before running the sample.
+   */
+  // const projectId = 'YOUR_PROJECT_ID';
+  // const location = 'us-central1';
+  // const datasetId = 'YOUR_DATASET_ID';
+
+  // Imports the Google Cloud AutoML library
+  const {AutoMlClient} = require('@google-cloud/automl').v1beta1;
+
+  console.log(operationFullId);
+  // Instantiates a client
+  const client = new AutoMlClient();
+
+  async function cancelOperation() {
+    // Construct request
+    const request = {
+      name: operationFullId,
+    };
+
+    client.operationsClient.cancelOperation({
+      name: operationFullId.toString(),
```

Comment:
Huh, what format do you expect this in?  If it's a string, don't think you need to `toString()` it